core/src/main/resources/hudson/model/Messages_zh_CN.properties

```diff
@@ -50,22 +249,172 @@ ResultTrend.StillUnstable=\u4ECD\u7136\u4E0D\u7A33\u5B9A
+UpdateCenter.Status.ConnectionFailed=\
+  <span class=error>\u65e0\u6cd5\u8fde\u63a5 {0}. \
+  \u3002\u4f60\u53ef\u80fd\u9700\u8981 <a href="../pluginManager/advanced">\u914d\u7f6eHTTP\u4ee3\u7406\uff1f</a></span>
+#初始化更新中心
+UpdateCenter.init=\u521d\u59cb\u5316\u66f4\u65b0\u4e2d\u5fc3
+#Jenkins更新通知
+UpdateCenter.CoreUpdateMonitor.DisplayName=Jenkins\u66f4\u65b0\u901a\u77e5
+
+#安卓开发
+UpdateCenter.PluginCategory.android=\u5b89\u5353\u5f00\u53d1
+#构建工具
+UpdateCenter.PluginCategory.builder=\u6784\u5efa\u5de5\u5177
+#构建包装
+UpdateCenter.PluginCategory.buildwrapper=\u6784\u5efa\u5305\u88c5
+UpdateCenter.PluginCategory.cli=\u547d\u4ee4\u884c\u63a5\u53e3
+UpdateCenter.PluginCategory.cloud=\u4e91\u63d0\u4f9b\u5546
+#集群管理和分布式构建
+UpdateCenter.PluginCategory.cluster=\u96c6\u7fa4\u7ba1\u7406\u548c\u5206\u5e03\u5f0f\u6784\u5efa
+UpdateCenter.PluginCategory.database=\u6570\u636e\u5e93
+UpdateCenter.PluginCategory.deployment=\u5f00\u53d1
+UpdateCenter.PluginCategory.devops=DevOps
+UpdateCenter.PluginCategory.dotnet=.NET \u5f00\u53d1
+#外部工具集成
+UpdateCenter.PluginCategory.external=\u5916\u90e8\u5de5\u5177\u96c6\u6210
+#Groovy相关
+UpdateCenter.PluginCategory.groovy-related=Groovy\u76f8\u5173
+UpdateCenter.PluginCategory.ios=iOS \u5f00\u53d1
+#插件库（被其他插件使用）
+UpdateCenter.PluginCategory.library=\u63d2\u4ef6\u5e93\uff08\u88ab\u5176\u4ed6\u63d2\u4ef6\u4f7f\u7528\uff09
+UpdateCenter.PluginCategory.listview-column=\u89c6\u56fe\u5217
+UpdateCenter.PluginCategory.maven=Maven
+#杂项
+UpdateCenter.PluginCategory.misc=\u6742\u9879
+UpdateCenter.PluginCategory.notifier=\u6784\u5efa\u901a\u77e5
+UpdateCenter.PluginCategory.page-decorator=\u9875\u9762\u88c5\u9970\u5668
+UpdateCenter.PluginCategory.parameter=\u6784\u5efa\u53c2\u6570
+#其他构建后动作
+UpdateCenter.PluginCategory.post-build=\u5176\u4ed6\u6784\u5efa\u540e\u52a8\u4f5c
+UpdateCenter.PluginCategory.python=Python \u5f00\u53d1
+UpdateCenter.PluginCategory.report=\u6784\u5efa\u62a5\u544a
+UpdateCenter.PluginCategory.ruby=Ruby \u5f00\u53d1
+#用于运行条件的插件
+UpdateCenter.PluginCategory.runcondition=\u7528\u4e8e\u8fd0\u884c\u6761\u4ef6\u7684\u63d2\u4ef6
+UpdateCenter.PluginCategory.scala=Scala \u5f00\u53d1
+#源码管理
+UpdateCenter.PluginCategory.scm=\u6e90\u7801\u7ba1\u7406
+#源码管理相关
+UpdateCenter.PluginCategory.scm-related=\u6e90\u7801\u7ba1\u7406\u76f8\u5173
+UpdateCenter.PluginCategory.security=\u5b89\u5168
+#代理启动器和控制器
+UpdateCenter.PluginCategory.slaves=\u4ee3\u7406\u542f\u52a8\u5668\u548c\u63a7\u5236\u5668
+#测试
+UpdateCenter.PluginCategory.test=\u6d4b\u8bd5
+UpdateCenter.PluginCategory.trigger=\u6784\u5efa\u89e6\u53d1\u5668
+UpdateCenter.PluginCategory.ui=\u7528\u6237\u754c\u9762
+UpdateCenter.PluginCategory.upload=\u6210\u54c1\u4e0a\u4f20
+UpdateCenter.PluginCategory.user=\u8ba4\u8bc1\u548c\u7528\u6237\u7ba1\u7406
+UpdateCenter.PluginCategory.view=\u89c6\u56fe
+UpdateCenter.PluginCategory.must-be-labeled=\u672a\u5206\u7c7b
+UpdateCenter.PluginCategory.unrecognized=Misc ({0})
```

Comment:
Not sure you need localization entry in such case

Reply:
I already saw the page, it's fine after the jenkins running.